src/OmniSharp.Abstractions/Options/CodeActionOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace OmniSharp.Options
+{
+    public class CodeActionOptions
+    {
+        public string[] LocationPaths { get; set; }
```

Comment:
It's kind of odd that there's a public, settable `LocationPaths` property that you're really not supposed to call and there's a `GetLocations(..)` method below that's the one you're really supposed to call.